src/binary/ibc-relayer/setup-prometheus.ts

```diff
@@ -0,0 +1,58 @@
+import http from 'http';
+
+import client from 'prom-client';
+
+import { Logger } from '../create-logger';
+
+let initialized = false;
+
+function getMetrics() {
+  return {
+    pollCounter: new client.Counter({
+      name: 'poll_counter',
+      help: 'Poll counter',
+    }),
+  };
+}
+
+export type Metrics = ReturnType<typeof setupPrometheus>;
```

Comment:
Why not define the type here and use it in the function definition?

Reply:
I agree it's safer than relying on the inference.

af3e18c